### PR TITLE
Upgrade dependencies

### LINKED ISSUE #472: Bump dependencies

### PR DESCRIPTION
**Pull request by dan13ram**
Also fixes eslint warnings
Also prettier to format all files
Also removes `metamaps` package
Also upgrades node version to 14.x

I'm doing no changes to actual logic.
If necessary I'd be happy to split this into multiple PRs.

Closes #472

### CODE REVIEW

**Comment by dan13ram** [comment]
@alalonde can you help me debug the deployment on vercel?

**Comment by dan13ram** [comment]
This was definitely not as straightforward or simple as I thought it would be. But I've managed to upgrade as many dependencies as possible. 
I've kept the version of `react` and other react based packages such as `react-dom` in the same version cause that would require a bunch more changes and testing.

Locally I've tested and the following works properly:
- bringing up storybook with proper chakra theme
- bringing up docker containers with backend, hasura and postgres along with migrateSourceCred accounts action
- bringing up frontend and editing/updating profile connected to both local backend as well as prod backend

**Review by alalonde — COMMENTED** [review]
Nice! You could probably split out the docs updates. Would you say the majority of this is refactoring / prettier-ing ?

**Comment by dan13ram** [comment]
> Nice! You could probably split out the docs updates. Would you say the majority of this is refactoring / prettier-ing ?

Yes it's ONLY refactoring and prettier-ing along with upgrading as many dependencies as possible. 
What exactly do you mean by "docs updates"?

**Comment by dan13ram** [comment]
The bulk of the changes are automatically generated by running `yarn lint --fix && yarn format`

File: packages/backend/package.json [hunk 1]
```diff
@@ -29,11 +29,11 @@
     "express-graphql": "0.12.0",
     "graphql": "15.5.0",
     "graphql-request": "3.4.0",
-    "graphql-tag": "2.11.0",
+    "graphql-tag": "2.12.4",
     "graphql-tools": "7.0.4",
     "imgix-core-js": "2.3.2",
     "node-fetch": "2.6.1",
-    "sourcecred": "0.8.4",
+    "sourcecred": "0.8.5",
```
Comment (alalonde) [inline packages/backend/package.json:36]: make sure you test this, we've encountered a number of breaking changes in recent sourcecred updates

**Review by alalonde — APPROVED** [review]
> The bulk of the changes are automatically generated by running `yarn lint --fix && yarn format`

Cool, good to know

File: packages/backend/package.json (hunk 1 above)
Comment (dan13ram) [reply packages/backend/package.json:36]: yes, it mostly adds functionality keeping the old stuff the same.

**Pull request merged**